src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Account.ConfirmEmailChange.cs.cshtml

```diff
@@ -34,9 +34,9 @@ namespace @thisNamespace
     public class ConfirmEmailChangeModel : PageModel
     {
         private readonly UserManager<@(Model.UserClass)> _userManager;
-        private readonly SignInManager<@(Model.UserClass> _signInManager;
+        private readonly SignInManager<@(Model.UserClass)> _signInManager;
```

Comment:
Can you update the version number in the branch to 3.1.1 servicing before taking this change?

Reply:
should be good now